<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commande</class>
 <widget class="QWidget" name="commande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="90,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButton_camera3">
               <property name="text">
                <string>Caméra 3</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QPushButton" name="pushButton_camera1">
               <property name="text">
                <string>Caméra 1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="images.qrc">:/images/images/blimp_dessus.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="margin">
                <number>-2</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QPushButton" name="pushButton_camera4">
               <property name="text">
                <string>Caméra 4</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QPushButton" name="pushButton_camera2">
               <property name="text">
                <string>Caméra 2</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_17"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QSlider" name="horizontalSlider">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>521</width>
            <height>19</height>
           </rect>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>20</y>
            <width>251</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="geometry">
           <rect>
            <x>750</x>
            <y>20</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_Retour">
          <property name="geometry">
           <rect>
            <x>1060</x>
            <y>20</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Retour</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
